File: coreServer/src/main/java/com/example/coreServer/dto/messageDto/AttachmentDto.java
package com.example.coreServer.dto.messageDto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AttachmentDto {
    private String id;
    private String type;
    private String url;
    private long sizeBytes;
    private String mimeType;
}

File: CSMiddleware/Shared/DTOs/ChatDTOs/AttachmentDTO.cs
using System;

namespace DTOs.ChatDTOs;

public class AttachmentDTO {
    public required String Id { get; set; }
    public required String Type { get; set; }
    public required String Url { get; set; }
    public required long SizeBytes { get; set; }
    public required String MimeType { get; set; }

}
